proposals/templates/proposals/proposal_copy.html

```diff
@@ -1,8 +1,6 @@
 {% extends "base/base.html" %}
-
```

Comment:
I think we should set `max-blank-lines=3` or something reasonable. I still want to be able to add empty lines for readability

https://djlint.com/docs/configuration/#max-blank-lines

Reply:
Done; I've also added some config to require a blank line after the extend and load 'blocks'